docs/home/SECURITY.md

```diff
@@ -16,7 +16,8 @@ antivirus/antimalware solution.
 
 ### Users and roles created by epiphany
 
-By default Epiphany is creating user operations that is used to connect to machines with admin rights on every machine. This setting can be changed in Epiphany yaml configuration files.
+By default, Epiphany creates user operations that is used to connect to machines with admin rights on every machine.
+This setting can be changed in Epiphany yaml configuration files.
```

Comment:
```suggestion
This setting can be changed in Epiphany initial yaml manifest.
```
Did you mean that?

Reply:
This should be done at configuration level.